feature-libs/checkout/base/styles/components/_promotions.scss

```diff
@@ -0,0 +1,21 @@
+cx-promotions {
```

Comment:
Please make sure to carry-over the changes from this PR: https://github.com/SAP/spartacus/commit/22c6ed6d3b3144cfb79391f8cb364cef54de84ee#diff-ea0c8f141c9d532f44b37a9c684982238886323a783858bd5041617778702490